docs/edge/monitoring.rst

```diff
@@ -0,0 +1,250 @@
+
+.. _edge-monitoring-deployment:
+
+Cluster Deployment
+------------------
+
+Deployment steps will depend on your :ref:`cloud provider <edge-providers>`,
+or on your local environment, if you choose the :ref:`self-hosted
+<edge-self-hosted>` variant of CrateDB Edge.
+
+.. _edge-monitoring-grafana-install:
+
+Grafana Install
+---------------
+
+The first important step comes after the successful installation of CrateDB into your
+chosen system. Right after you get the following console prompt:
+
+.. code-block:: console
+
+  ✓ Region is connected to CrateDB Cloud.
+  ✓ Successfully validated installation
+
+This means that base installation is done. You will now be asked if you want to
+have Grafana installed in the system. Press ``Y`` to opt-in:
+
+.. code-block:: console
+
+  Optional installation of Loki and Grafana
+  -----------------------
+  ! Loki and Grafana not detected.
+ 
+  =============================================================================
+  CrateDB Edge encourages the usage of Loki and Grafana to ease support.
+  Would you like to install Loki and Grafana?
+ 
+  =============================================================================
+  Install? [Yy] y
+
+After successful installation, you will get the following output:
+
+.. code-block:: console
+
+  ✓ Installed.
+  Loki and Grafana successfully installed.
+
+.. _edge-monitoring-accessing-grafana:
+
+Accessing Grafana UI
+--------------------
+
+You will notice that during the installation of Grafana, you are provided with
+a number of useful commands that will help you access the Grafana UI once its
+installation is finished.
+
+.. _edge-monitoring-grafana-password:
+
+Grafana user password
+'''''''''''''''''''''
+
+As a part of the installation process, a user account is created for Grafana. The
+username is ``admin`` and the password can be retrieved with the following
+command:
+
+.. code-block:: console
+  
+  kubectl get secret --namespace crate-loki crate-grafana -o jsonpath="{.data.admin-password}" | base64 --decode ; echo
+
+You will get a response with a password:
+
+.. code-block:: console
+
+  rpz654Q2k8D5vgcvvrRgQMZAC2gKxZJH4oQf09fB
+
+.. _edge-monitoring-port-forwarding:
+
+Port-forwarding to get the Grafana URL
+''''''''''''''''''''''''''''''''''''''
+
+To get the URL of your Grafana UI, you need to issue the following commands:
+
+.. code-block:: console
+
+  export POD_NAME=$(kubectl get pods --namespace crate-loki -l "app.kubernetes.io/name=grafana,app.kubernetes.io/instance=crate-grafana" -o jsonpath="{.items[0].metadata.name}")
+  
+  kubectl --namespace crate-loki port-forward $POD_NAME 3000
+
+This will forward your running Grafana instance to port ``3000``. You can then
+view it on that port. For this tutorial, we used Google Cloud Engine. After 
+issuing the command, we use the "Web preview" to see Grafana UI on port 3000.
+
+.. image:: ../_assets/img/edge-monitoring-forwarding.png
+   :alt: Google Cloud web preview
+
+.. NOTE::
+
```

Comment:
Do you think we need this section? Port usage does not depend on your cloud provider - merely on what you've currently got running. Maybe not worth elaborating on this section, as we don't cover other possible error scenarios elsewhere...

Reply:
I'd be more than happy to get rid of it honestly, it's way too long for a note, looks terrible. The thing is, I think it can happen. I was doing all testing on GKE. I tried multiple CPU tiers, multiple Kubernetes versions. Sometimes the `:3000` was free, sometimes occupied in a fresh GKE cluster. Should I just remove it?

Reply:
Talked with Roman too, he agrees with you. Will edit.